frontend/app/routes/$lang+/_public+/apply+/$id+/personal-information.tsx

```diff
@@ -386,162 +386,168 @@ export default function ApplyFlowPersonalInformation() {
             />
           </div>
           <h2 className="mb-4 font-lato text-2xl font-bold">{t('apply:personal-information.mailing-address.header')}</h2>
-          <div className="my-6 space-y-6">
-            <InputField
-              id="mailing-address"
-              name="mailingAddress"
-              className="w-full"
-              label={t('apply:personal-information.address-field.address')}
-              maxLength={30}
-              helpMessagePrimary={t('apply:personal-information.address-field.address-note')}
-              helpMessagePrimaryClassName="text-black"
-              autoComplete="address-line1"
-              defaultValue={defaultState?.mailingAddress ?? ''}
-              errorMessage={errorMessages['mailing-address']}
-              required
-            />
-            <InputField
-              id="mailing-apartment"
-              name="mailingApartment"
-              className="w-full"
-              label={t('apply:personal-information.address-field.apartment')}
-              maxLength={30}
-              autoComplete="address-line2"
-              defaultValue={defaultState?.mailingApartment ?? ''}
-              errorMessage={errorMessages['mailing-apartment']}
-              required
-            />
-            <InputSelect
-              id="mailing-country"
-              name="mailingCountry"
-              className="w-full sm:w-1/2"
-              label={t('apply:personal-information.address-field.country')}
-              autoComplete="country"
-              defaultValue={defaultState?.mailingCountry ?? ''}
-              errorMessage={errorMessages['mailing-country']}
-              options={[dummyOption, ...countries]}
-              onChange={mailingCountryChangeHandler}
-              required
-            />
-            {mailingRegions.length > 0 && (
-              <InputSelect
-                id="mailing-province"
-                name="mailingProvince"
-                className="w-full sm:w-1/2"
-                label={t('apply:personal-information.address-field.province')}
-                defaultValue={defaultState?.mailingProvince ?? ''}
-                errorMessage={errorMessages['mailing-province']}
-                options={[dummyOption, ...mailingRegions]}
-                required
-              />
-            )}
-            <div className="grid items-end gap-6 md:grid-cols-2">
+          <fieldset>
+            <legend className="sr-only">{t('apply:personal-information.mailing-address.header')}</legend>
```

Comment:
`legend` should have the current classes of the `h2` and the `h2` should be remove because it's replaced by the fieldset `legend`. Do the same for the other `fieldset`.

Reply:
> `legend` should have the current classes of the `h2` and the `h2` should be remove because it's replaced by the fieldset `legend`. Do the same for the other `fieldset`.

@sebastien-comeau 
Updated.
But I think this should be a temporary fix since we don't have an actual legend text in the design. I'm not sure is using legend as heading is a good practice, legend should be a descriptive text for the field.
Anyway, we can come back and update it if they update the design in the future.

Reply:
>The `<fieldset>` tag is used to group related elements in a form.
https://www.w3schools.com/tags/tag_fieldset.asp